docs/_ext/djangodocs.py

```diff
@@ -254,10 +254,7 @@ def visit_console_html(self, node):
         self.body.append('<section class="c-content-win" id="c-content-%(id)s-win">\n' % {'id': uid})
         win_text = node['win_console_text']
         highlight_args = {'force': True}
-        if 'linenos' in node:
-            linenos = node['linenos']
-        else:
-            linenos = win_text.count('\n') >= self.highlightlinenothreshold - 1
```

Comment:
Is it always `False`?

Reply:
`win_text.count('\n') >= self.highlightlinenothreshold - 1` is practically always false because `self.highlightlinenothreshold` is a property that returns `sys.maxsize` (cf https://github.com/sphinx-doc/sphinx/commit/61098a0ae2e696a804459d36bd74ca57db76eda5#diff-9ae39fbf16dabc27af998b9c0a6968a0L973)

Reply:
:thinking: so in some rare cases this can be `True`, maybe we should just restore the previous logic `linenos = win_text.count('\n') >= sys.maxsize - 1`.

@ramiro What do you think?

Reply:
`sys.maxsize` is 9223372036854775807 on my computer. If you get that many `\n` characters in a paragraph, I think something has gone seriously wrong :laughing: .

Reply:
Sphinx's own code uses the same approach: https://github.com/sphinx-doc/sphinx/blob/master/sphinx/writers/html.py#L476

Reply:
Fine, you persuaded me :smile:  :speaker: